bin/bin.js

```diff
@@ -287,6 +289,22 @@ const main = async () => {
       }
     }
 
+    if (exec) {
+      const execCmd = exec
+        .replace("{}", _path.resolve(outputPodcastPath))
+        .replace(
+          "{filenameBase}",
+          _path.resolve(
+            episodeFilename.substring(0, episodeFilename.lastIndexOf("."))
+          )
+        );
+      child.exec(execCmd).on("exit", (code) => {
```

Comment:
I would like to keep the script synchronous for now. Could we swap this to `child.execSync`?

Reply:
Sure good idea, see 008babf